docs/content/community/bijeenkomsten/20181025-demo-sprint-5.md

```diff
@@ -12,8 +12,39 @@ Amsterdam, 25 oktober 2018
 Slides:
 
 * [ZDS2 Demo Spint 5.pdf](/community/bestanden/zds2-demo-sprint-5.pdf)
-* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf))
+* [Opsplitsen RGBZ](/community/bestanden/opsplitsen-rgbz.pdf)
 
 ## Feedback
 
-[aanvullen]
+### Wat is de rol en scope van mapper service? Moet dit gestandaardiseerd worden? ###
+todo
+
+### Hoe staat het met authenticatie en autorisatie? ###
+todo
+
+### Hoe gaan jullie om met externe gebruikers m.b.t. authenticatie en autorisatie? ###
+todo
+
+### De URL werkt niet bij update!? ###
+todo
+
+### Wat is er concreet op 20 december en wat is het vervolg? Stip op de horizon? ###
+todo
+
+### Wordt Docker veel gebruikt binnen gemeenten? ###
+todo
+
+### Wat is de huidige toepasbaarheid van de API's? ###
+todo
```

Comment:
Lastig te beantwoorden. Het is flauw om te zeggen: MOR, Straatartiest, etc. maar weet niet zo goed een antwoord te formuleren.

Reply:
Ik denk eigenlijk dat de vraag anders bedoeld is. De huidige API's zijn erg low-level. Alles is mogelijk maa het maakt het voor ontwikkelaars lastiger (want meer kennis vereist) om ze toe te passen dan wanneer er een proces/experience/convenience  API-laag bovenop geboden wordt. 

Zie ook (alos voorbeeld): https://blogs.mulesoft.com/wp-content/uploads/2017/07/api-led-connectivity.png

Sommige acties als het aanmaken van een zaak hoeven denk ik niet echt in een hogere laag opgenomen te worden, dat is gewoon een POST op de zaken api. Maar bijvoorbeeld het toevoegen van een informatieobject en het relateren aan een zaak (twee acties) zou je kunnen combineren tot een enkele call waarbij je zelfs in de logica van die api nog in zou kunnen bouwen dat ook het relateren in de ZRC uitgevoerd wordt.

Door veel gebruikte combinaties van calls (die we misschien nu nog niet weten omdat de API nog  niet in productie is) aan te bieden als proces/experience API wordt het gebruiksgemak en daarmee de toepasbaarheid denk ik wel vergroot. 

Een eerste aanzet tot een proces API kan al zijn het uitschrijven van de toepassingspatronen (wanneer gebruik je welke call), een soort recepten uit een kookboek om dingen voor elkaar te krijgen. Bij voldoende behoefte kan altijd nog besloten worden een proces of experience API uit te werken.

Reply:
Ik heb een gecondenseerd antwoord opgenomen die beide reacties meeneemt